--- a/DNS-SERVER-MIB
+++ b/DNS-SERVER-MIB
@@ -2,7 +2,7 @@
 
    IMPORTS
        mib-2
-           FROM RFC-1213
+        FROM RFC1213-MIB
        MODULE-IDENTITY, OBJECT-TYPE, OBJECT-IDENTITY,
        IpAddress, Counter32, Gauge32
            FROM SNMPv2-SMI
@@ -11,11 +11,6 @@
        MODULE-COMPLIANCE, OBJECT-GROUP
            FROM SNMPv2-CONF;
 
-   dns OBJECT-IDENTITY
-       STATUS  current
-       DESCRIPTION
-               "The OID assigned to DNS MIB work by the IANA."
-       ::= { mib-2 32 }
 
    dnsServMIB MODULE-IDENTITY
        LAST-UPDATED "9401282251Z"
@@ -43,6 +38,11 @@
                "The MIB module for entities implementing the server side
                of the Domain Name System (DNS) protocol."
        ::= { dns 1 }
+dns OBJECT-IDENTITY
+    STATUS  current
+    DESCRIPTION
+            "The OID assigned to DNS MIB work by the IANA."
+    ::= { mib-2 32 }
    dnsServMIBObjects       OBJECT IDENTIFIER ::= { dnsServMIB 1 }
 
    -- (Old-style) groups in the DNS server MIB.
@@ -101,7 +101,7 @@
        SYNTAX  DnsName
 
    DnsClass ::= TEXTUAL-CONVENTION
-       DISPLAY-HINT "2d"
+    DISPLAY-HINT "d"
        STATUS  current
        DESCRIPTION
                "This data type is used to represent the class values
@@ -114,7 +114,7 @@
        SYNTAX  INTEGER (0..65535)
 
    DnsType ::= TEXTUAL-CONVENTION
-       DISPLAY-HINT "2d"
+    DISPLAY-HINT "d"
        STATUS  current
        DESCRIPTION
                "This data type is used to represent the type values
@@ -127,7 +127,7 @@
        SYNTAX  INTEGER (0..65535)
 
    DnsQClass ::= TEXTUAL-CONVENTION
-       DISPLAY-HINT "2d"
+    DISPLAY-HINT "d"
        STATUS  current
        DESCRIPTION
                "This data type is used to represent the QClass values
@@ -139,7 +139,7 @@
                "RFC-1035 section 3.2.5."
        SYNTAX  INTEGER (0..65535)
    DnsQType ::= TEXTUAL-CONVENTION
-       DISPLAY-HINT "2d"
+    DISPLAY-HINT "d"
        STATUS  current
        DESCRIPTION
                "This data type is used to represent the QType values
@@ -152,7 +152,7 @@
        SYNTAX  INTEGER (0..65535)
 
    DnsTime ::= TEXTUAL-CONVENTION
-       DISPLAY-HINT "4d"
+    DISPLAY-HINT "d"
        STATUS  current
        DESCRIPTION
                "DnsTime values are 32-bit unsigned integers which
