--- a/INTEGRATED-SERVICES-MIB
+++ b/INTEGRATED-SERVICES-MIB
@@ -4,7 +4,7 @@
             MODULE-IDENTITY, OBJECT-TYPE, Counter32,
             Gauge32, Integer32, mib-2                FROM SNMPv2-SMI
             TimeInterval, TEXTUAL-CONVENTION, RowStatus,
-            TruthValue                               FROM SNMPv2-TC
+            TruthValue, TestAndIncr                  FROM SNMPv2-TC
             MODULE-COMPLIANCE, OBJECT-GROUP          FROM SNMPv2-CONF
             ifIndex, InterfaceIndex                  FROM IF-MIB;
 
@@ -84,7 +84,7 @@
        SYNTAX   INTEGER (1..255)
 
     Port ::= TEXTUAL-CONVENTION
-        DISPLAY-HINT "d"
+        -- DISPLAY-HINT "d" ---- illegal for OCTET STRING.
         STATUS   current
         DESCRIPTION
            "The value of the UDP or TCP Source or Destina-
