--- a/MIP-MIB
+++ b/MIP-MIB
@@ -2,12 +2,13 @@
 
     IMPORTS
         Counter32, Gauge32, Integer32, IpAddress, experimental,
-        MODULE-IDENTITY, OBJECT-TYPE, NOTIFICATION-TYPE
+        MODULE-IDENTITY, OBJECT-TYPE, NOTIFICATION-TYPE, mib-2,
+        Unsigned32
                                         FROM SNMPv2-SMI
         RowStatus, TruthValue, TimeStamp,
         TEXTUAL-CONVENTION
                                         FROM SNMPv2-TC
-        MODULE-COMPLIANCE, OBJECT-GROUP
+        MODULE-COMPLIANCE, OBJECT-GROUP, NOTIFICATION-GROUP
                                         FROM SNMPv2-CONF;
 
     mipMIB    MODULE-IDENTITY
@@ -258,7 +259,7 @@
 
     mipSecViolatorAddress  OBJECT-TYPE
         SYNTAX      IpAddress
-        MAX-ACCESS  accessible-for-notify
+        MAX-ACCESS  read-only
         STATUS      current
         DESCRIPTION
                 "Violator's IP address. The violator is not necessary
@@ -2096,7 +2097,7 @@
                 function within a home agent."
         ::= { mipGroups 12 }
 
-    mipSecNotifcationsGroup NOTIFICATION-GROUP
+    mipSecNotificationsGroup NOTIFICATION-GROUP
         NOTIFICATIONS { mipAuthFailure }
         STATUS      current
         DESCRIPTION
