--- a/RFC1316-MIB
+++ b/RFC1316-MIB
@@ -3,7 +3,7 @@
                     IMPORTS
                             Counter, TimeTicks, Gauge
                                     FROM RFC1155-SMI
-                            DisplayString
+                DisplayString, mib-2
                                     FROM RFC1213-MIB
                             OBJECT-TYPE
                                     FROM RFC-1212;
